root/main/index.js

```diff
@@ -16,6 +16,7 @@ import {CONTACT_URL} from '../static/scripts/common/constants';
 import {reduceArtistCredit}
   from '../static/scripts/common/immutable-entities';
 import entityHref from '../static/scripts/common/utility/entityHref';
+import AppDownload from '../components/Home/AppDownload';
```

Comment:
Minor, but - we usually have lowercase folders, so I'd expect the same here. Either /home or /homepage seems fine to me.

Reply:
Done